cdxev/__main__.py

```diff
@@ -709,6 +711,40 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of a SBOM for manual completion"),
+    )
+    parser.add_argument(
+        "--name",
+        metavar="<schema path>",
```

Comment:
wrong metavar, it's used within the help text, I don't think that was intentional:
![image](https://github.com/user-attachments/assets/c0b60d71-9e70-46a3-a169-eb587b02a6e6)

Reply:
adapted